bundle/Controller/AccessController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Netgen\TagsBundle\Controller;
```

Comment:
This controller should go to `Admin` namespace and extend the base controller from that namespace.